enterprise/coderd/coderd.go

```diff
@@ -352,6 +352,7 @@ func New(ctx context.Context, options *Options) (_ *API, err error) {
 				r.Use(
 					httpmw.ExtractProvisionerKeyParam(options.Database),
 				)
+				r.Get("/tags", api.fetchProvisionerKeyTags)
```

Comment:
review: Just seen that when using provisioner key instead of user auth - it seems we can not attach an organization to the call - I plan to remove the organizations part from the url wdyt ?
Instead have : `/api/v2/provisionerskeys/tags` taht will use the provisioner key only.

Reply:
I think that's fair; the provisioner key is scoped to an org anyway.